srcpkgs/linux-wifi-hotspot/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'linux-wifi-hotspot'
+pkgname=linux-wifi-hotspot
+version=4.5.0
+revision=1
+archs="i686 x86_64 x86_64-musl"
+build_style=gnu-configure
+hostmakedepends="pkg-config which glib-devel"
+makedepends="gtk+3-devel libpng-devel qrencode-devel"
+depends="gtk+3 bash dnsmasq hostapd iproute2 iptables iw procps-ng util-linux"
+short_desc="Feature-rich wifi hotspot creator"
+maintainer="Bnyro <bnyro@tutanota.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/lakinduakash/linux-wifi-hotspot"
+distfiles="https://github.com/lakinduakash/linux-wifi-hotspot/archive/refs/tags/v${version}.tar.gz"
+checksum=9e4222b7d3129347b01eef711911ec87c9a04e20efd12d88565f682a14eb91da
+conflicts="create_ap"
+
+do_configure() {
+	:
```

Comment:
Not sure why you use the `gnu-configure` build style and set an empty `do_configure`. This should probably use `build_style=gnu-makefile` instead.

Reply:
When using gnu-makefile as build style, the builds failed with the reason that <gtk/gtk.h> wasn't found. Somehow it works with gnu-configure.